src/pages/GroupPage.jsx

```diff
@@ -1,12 +1,36 @@
+import { useParams } from "react-router-dom";
+
+import asyncGetUserGroup from "../api/group/asyncGetUserGroup";
 import DashboardHeader from "../components/Header/DashboardHeader";
 import DashboardSidebar from "../components/Sidebar/DashboardSidebar";
+import useNoSignInRedirect from "../hooks/useNoSignInRedirect";
+import useBoundStore from "../store/client/useBoundStore";
+import { useQuery } from "@tanstack/react-query";
 
 const GroupPage = () => {
+  const { groupId } = useParams();
+
+  useNoSignInRedirect();
```

Comment:
```suggestion
  useNoSignInRedirect();

  const { groupId } = useParams();
```
간단한 수정 제안 드려요.
저희의 의도는 '로그인 여부 검사가 먼저 진행' 되는 것이라, 동작에 차이는 없지만 순서를 조정해보면 어떨까요? `useNoSignInRedirect` 훅을 사용하는 컴포넌트에서는 항상 요 훅을 최상단에서 호출하는 규칙을 적용해보면 좋겠습니다 :)

Reply:
가독성을 위해 좋은 제안이신 것 같습니다 :) 바로 적용하겠습니다